src/setupTests.js

```diff
@@ -2,4 +2,6 @@
 // allows you to do things like:
 // expect(element).toHaveTextContent(/react/i)
 // learn more: https://github.com/testing-library/jest-dom
-import "@testing-library/jest-dom/extend-expect";
+import "@testing-library/jest-dom/extend-expect"
+
+jest.mock("react-ga")
```

Comment:
What is the purpose of this?

Reply:
I saw some errors regarding google analytics while running test cases, so added mock for whole package. Just to reduce the console errors and minimise the output of test suite.